src/i18n/pl-PL/index.ts

```diff
@@ -13,14 +13,14 @@ export default {
     },
     proxy: {
       name: 'Atakuj tylko z PROXY',
-      description: 'Bez tej opcji program będzie używał Twojego IP. Jest to o wiele bardziej wydajne, Twoje ataki są mocniejsze, ale wróg poznaje Twój adres IP i nie będziesz anonimowy.'
+      description: 'Bez tej opcji program będzie używał Twojego IP. Jest to o wiele bardziej wydajne, Twoje ataki są mocniejsze, ale wróg poznaje Twój adres IP i nie będziesz anonimowy'
     },
     advanced: {
       header: 'Zaawansowane ustawienia',
       description: 'Jeżeli nie jesteś doświadczonym użytkownikiem nie zmieniaj ustawień zaawansowanych. Może to mieć zły wpływ na działanie programu',
       masDosersCount: {
         name: 'Рівень паралельності (TODO)',
-        description: 'Задає кількість паралельних процесів. Використовуються асинхронічні механізми (не потокові), тому це не сильно впливає на CPU/RAM. Наразі опція діє в ручному режимі і задає кількість паралельних запитів. При великій кількості запитів, проксі сервера можуть блокувати DDOS. Змінюйте вартість і чекайте 2-3 хвилини, щоб побачити результат. Якщо бачите ECONNABORTED - це значить Вас блокують і треба зменшити кількість.'
+        description: 'Задає кількість паралельних процесів. Використовуються асинхронічні механізми (не потокові), тому це не сильно впливає на CPU/RAM. Наразі опція діє в ручному режимі і задає кількість паралельних запитів. При великій кількості запитів, проксі сервера можуть блокувати DDOS. Змінюйте вартість і чекайте 2-3 хвилини, щоб побачити результат. Якщо бачите ECONNABORTED - це значить Вас блокують і треба зменшити кількість'
```

Comment:
Określa liczbę równoległych procesów.
Stosowane są mechanizmy asynchroniczne (nie strumieniowe), więc nie ma to dużego wpływu na procesor / RAM.
Obecnie opcja działa w trybie manualnym i określa liczbę równoległych zapytań.
Przy dużej liczbie requestów proxy serwery mogą blokować DDOS.
Zmień znaczenie i poczekaj 2-3 minuty, aby zobaczyć wynik.
Jeśli widzisz ECONNABORTED - oznacza to, że jesteś zablokowany i musisz zmniejszyć liczbę/ilość.

Reply:
Dziękuję